src/components/AreaSelector/AreaSelector.js

```diff
@@ -5,10 +5,18 @@ function AreaSelector({regionNames, onChange, currentRegion}) {
   var valueOptions = [];
   regionNames.forEach(name => valueOptions.push({"value": name, "label": name}));
 
+  const customStyles = {
```

Comment:
What was your reasoning behind doing it this way instead of using CSS?

Reply:
@corviday , @helfy18 , if I may offer some thoughts: 
- That selector (from pcic-react-components) is, at root, a [React Select](https://react-select.com/home) selector. 
- React Select [works in the way Johnathan coded it.](https://react-select.com/styles), namely with a function like that. J may have copied it from pre-existing code I wrote.
- That, in turn, is the [emotion](https://emotion.sh/docs/introduction) library, which I've never used outside of React Select. It doesn't seem insane on first inspection, but that doesn't mean for sure that it's not :).
- I believe this is the right way to style React Select -- it's what the package uses, and we shouldn't try to end run around it if we don't have to, and I don't think we have to.

Reply:
nvm, I see that the PR has already been merged. My comments may however be useful in future.

Reply:
@rod-glover thanks for your input! I hadn't seen that sort of styling before and while I verified that it worked, I wasn't sure it was the _right_ way. Glad to hear it was.